vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
@Fryguy, @blomquisg how do you debug this with the suggested syntax if you want to understand the result/return value ? I mean it takes away the local variable that you can evaluate, so how do u debug in this case the method (not via the calling code)?